Library/Formula/symlinks.rb

```diff
@@ -0,0 +1,23 @@
+class Symlinks < Formula
+  desc "scan/change symbolic links"
+  homepage "http://www.ibiblio.org/pub/Linux/utils/file/symlinks.lsm"
```

Comment:
That wouldn't make sense as far as I can tell. Quickly grepping through existing formulae I see that most (if not all) occurrences of `MacOS.sdk_path` are in places that do not respond to `-isysroot` but require the full path. But passing a path via `-I` is different as it will respect `-isysroot`. (I have no system without the CLT, so cannot confirm this experimentally.)

But if this approach is really causing too many headaches, we could as well use `inreplace` on `symlinks.c` to change `#include <malloc.h>` into `#include <malloc/malloc.h>`. (The end result will be the same.)